app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/ExpandPhotoActivity.kt

```diff
@@ -87,6 +108,27 @@ class ExpandPhotoActivity : AppCompatActivity() {
         }
     }
 
+    /**
+     * Called when toolbar's "delete" is pressed
+     */
+    private fun onDeletePressed() {
+        // get the id of the current item displayed in the ViewPager and delete it
+        LaunchNoteDatabase
+                .getDatabase(this)
+                ?.picNoteDao()
+                ?.findById(picNotes[viewPager.currentItem].id)
+                ?.map { picNotes ->
+                    picNotes.forEach {
+                        val success = PicNote.deleteFromDb(this, it)
+                        if (!success) Toast.makeText(this, "Cannot delete $defaultImageId", Toast.LENGTH_LONG).show()
+                    }
+                }?.subscribeOn(Schedulers.io())
+                ?.observeOn(AndroidSchedulers.mainThread())
+                ?.subscribe()
+        // exit full screen
+        super.onBackPressed()
```

Comment:
I know that this comment is not related to this commit  but, if its possible can we clear Glide's cache at least every time that we exit this view? It seem really inefficient to keep those massive objects lying around.